Modules/Core/Common/include/itkImageBase.hxx

```diff
@@ -384,17 +380,16 @@ ImageBase<VImageDimension>::VerifyRequestedRegion()
   const SizeType & requestedRegionSize = this->GetRequestedRegion().GetSize();
   const SizeType & largestPossibleRegionSize = this->GetLargestPossibleRegion().GetSize();
 
-  for (i = 0; i < VImageDimension; ++i)
+  for (unsigned int i = 0; i < VImageDimension; ++i)
   {
     if ((requestedRegionIndex[i] < largestPossibleRegionIndex[i]) ||
         ((requestedRegionIndex[i] + static_cast<OffsetValueType>(requestedRegionSize[i])) >
          (largestPossibleRegionIndex[i] + static_cast<OffsetValueType>(largestPossibleRegionSize[i]))))
     {
-      retval = false;
+      return false;
```

Comment:
In general I like the intentions of this PR, thanks Hans. 👍 However, you can still make it easier to review when you do just one kind of refactoring at a time. ("One Commit One Change".) Especially so when the commit is quite large (hundreds of changes.) Maybe something to consider for the next time....?